main.py

```diff
@@ -1,10 +1,39 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+
+    for player_name, player_data in players.items():
+        race, _ = Race.objects.get_or_create(
+            name=player_data["race"]["name"],
+            description=player_data["race"]["description"])
+
+        for skill_data in player_data["race"]["skills"]:
+            Skill.objects.get_or_create(
+                name=skill_data["name"],
+                bonus=skill_data["bonus"],
+                race=race)
+
+        guild_data = player_data["guild"] if player_data["guild"] else None
```

Comment:
You can simplify your code
```suggestion
        guild_data = player_data.get("guild")
```